tests/test_ingesters.py

```diff
@@ -174,8 +178,11 @@ class MetanormIngesterTestCase(django.test.TestCase):
     """Test the base metadata ingester class"""
 
     def setUp(self):
+        Parameter.objects.count = Mock()
+        Parameter.objects.count.return_value=2
```

Comment:
@aperrin66 
It was written for using tests considering this line:
https://github.com/nansencenter/django-geo-spaas-harvesting/blob/88beb9d559165eb202f37c3786f4a178f562a6c5/geospaas_harvesting/ingesters.py#L53

Reply:
You need to use `patch()` if you want that mock to work

Reply:
But it is working right now with this one! Any problem with it?

Reply:
It's not doing anything, if you remove it the tests will behave the same way

Reply:
Lets make the things ready in the metanorm repo before doing anything about the harvesting repo. and then we come back to these tests. In my laptop, since the metanorm is new, the tests does not have any problem. we will come back to this part if necessary.